docs/api/graphql_queries.md

```diff
@@ -43,6 +43,30 @@ Response:
 
 You can request any Fields of the Content item. In the example above, these are `title` and `author`.
 
+You can also query the generic `item` object. The `item` object references either a Content item or a Location.
```

Comment:
That is actually a bit incorrect: an item is mostly a content, and location data can be requested for it. If it was loaded using a locationId or an urlAlias, it will refer to that specific location. If it was loaded using a content id, the location will be guessed automatically.

The same thing is true for any specific type (article, folder...)